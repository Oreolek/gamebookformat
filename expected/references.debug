BEGIN DEBUG OUTPUT
Book title: Gamebook
Number of sections: 400
Introduction
 This gamebook demonstrates simple references between sections. Also notice that an intro section like this one can reference sections, like the start at 1 or the next section at 400. References can include an optional name that is used in some formats, so the same links could also be rendered as start (1) and next (400). 


Turn to 1 to begin.
1 (start) -  This is where the adventure begins. You can go on to the next section, see 400 or try the random selection feature at 399. 
2 () - (EMPTY)
3 () - (EMPTY)
4 () - (EMPTY)
5 () - (EMPTY)
6 () - (EMPTY)
7 () - (EMPTY)
8 () - (EMPTY)
9 () - (EMPTY)
10 () - (EMPTY)
11 () - (EMPTY)
12 () - (EMPTY)
13 () - (EMPTY)
14 () - (EMPTY)
15 () - (EMPTY)
16 () - (EMPTY)
17 () - (EMPTY)
18 () - (EMPTY)
19 () - (EMPTY)
20 () - (EMPTY)
21 () - (EMPTY)
22 () - (EMPTY)
23 () - (EMPTY)
24 () - (EMPTY)
25 () - (EMPTY)
26 () - (EMPTY)
27 () - (EMPTY)
28 () - (EMPTY)
29 () - (EMPTY)
30 () - (EMPTY)
31 () - (EMPTY)
32 () - (EMPTY)
33 () - (EMPTY)
34 () - (EMPTY)
35 () - (EMPTY)
36 () - (EMPTY)
37 () - (EMPTY)
38 () - (EMPTY)
39 () - (EMPTY)
40 () - (EMPTY)
41 () - (EMPTY)
42 () - (EMPTY)
43 () - (EMPTY)
44 () - (EMPTY)
45 () - (EMPTY)
46 () - (EMPTY)
47 () - (EMPTY)
48 () - (EMPTY)
49 () - (EMPTY)
50 () - (EMPTY)
51 () - (EMPTY)
52 () - (EMPTY)
53 () - (EMPTY)
54 () - (EMPTY)
55 () - (EMPTY)
56 () - (EMPTY)
57 () - (EMPTY)
58 () - (EMPTY)
59 () - (EMPTY)
60 () - (EMPTY)
61 () - (EMPTY)
62 () - (EMPTY)
63 () - (EMPTY)
64 () - (EMPTY)
65 () - (EMPTY)
66 () - (EMPTY)
67 () - (EMPTY)
68 () - (EMPTY)
69 () - (EMPTY)
70 () - (EMPTY)
71 () - (EMPTY)
72 () - (EMPTY)
73 () - (EMPTY)
74 () - (EMPTY)
75 () - (EMPTY)
76 () - (EMPTY)
77 () - (EMPTY)
78 () - (EMPTY)
79 () - (EMPTY)
80 () - (EMPTY)
81 () - (EMPTY)
82 () - (EMPTY)
83 () - (EMPTY)
84 () - (EMPTY)
85 () - (EMPTY)
86 () - (EMPTY)
87 () - (EMPTY)
88 () - (EMPTY)
89 () - (EMPTY)
90 () - (EMPTY)
91 () - (EMPTY)
92 () - (EMPTY)
93 () - (EMPTY)
94 () - (EMPTY)
95 () - (EMPTY)
96 () - (EMPTY)
97 () - (EMPTY)
98 () - (EMPTY)
99 () - (EMPTY)
100 () - (EMPTY)
101 () - (EMPTY)
102 () - (EMPTY)
103 () - (EMPTY)
104 () - (EMPTY)
105 () - (EMPTY)
106 () - (EMPTY)
107 () - (EMPTY)
108 () - (EMPTY)
109 () - (EMPTY)
110 () - (EMPTY)
111 () - (EMPTY)
112 () - (EMPTY)
113 () - (EMPTY)
114 () - (EMPTY)
115 () - (EMPTY)
116 () - (EMPTY)
117 () - (EMPTY)
118 () - (EMPTY)
119 () - (EMPTY)
120 () - (EMPTY)
121 () - (EMPTY)
122 () - (EMPTY)
123 () - (EMPTY)
124 () - (EMPTY)
125 () - (EMPTY)
126 () - (EMPTY)
127 () - (EMPTY)
128 () - (EMPTY)
129 () - (EMPTY)
130 () - (EMPTY)
131 () - (EMPTY)
132 () - (EMPTY)
133 () - (EMPTY)
134 () - (EMPTY)
135 () - (EMPTY)
136 () - (EMPTY)
137 () - (EMPTY)
138 () - (EMPTY)
139 () - (EMPTY)
140 () - (EMPTY)
141 () - (EMPTY)
142 () - (EMPTY)
143 () - (EMPTY)
144 () - (EMPTY)
145 () - (EMPTY)
146 () - (EMPTY)
147 () - (EMPTY)
148 () - (EMPTY)
149 () - (EMPTY)
150 () - (EMPTY)
151 () - (EMPTY)
152 () - (EMPTY)
153 () - (EMPTY)
154 () - (EMPTY)
155 () - (EMPTY)
156 () - (EMPTY)
157 () - (EMPTY)
158 () - (EMPTY)
159 () - (EMPTY)
160 () - (EMPTY)
161 () - (EMPTY)
162 () - (EMPTY)
163 () - (EMPTY)
164 () - (EMPTY)
165 () - (EMPTY)
166 () - (EMPTY)
167 () - (EMPTY)
168 () - (EMPTY)
169 () - (EMPTY)
170 () - (EMPTY)
171 () - (EMPTY)
172 () - (EMPTY)
173 () - (EMPTY)
174 () - (EMPTY)
175 () - (EMPTY)
176 () - (EMPTY)
177 () - (EMPTY)
178 () - (EMPTY)
179 () - (EMPTY)
180 () - (EMPTY)
181 () - (EMPTY)
182 () - (EMPTY)
183 () - (EMPTY)
184 () - (EMPTY)
185 () - (EMPTY)
186 () - (EMPTY)
187 () - (EMPTY)
188 () - (EMPTY)
189 () - (EMPTY)
190 () - (EMPTY)
191 () - (EMPTY)
192 () - (EMPTY)
193 () - (EMPTY)
194 () - (EMPTY)
195 () - (EMPTY)
196 () - (EMPTY)
197 () - (EMPTY)
198 () - (EMPTY)
199 () - (EMPTY)
200 () - (EMPTY)
201 () - (EMPTY)
202 () - (EMPTY)
203 () - (EMPTY)
204 () - (EMPTY)
205 () - (EMPTY)
206 () - (EMPTY)
207 () - (EMPTY)
208 () - (EMPTY)
209 () - (EMPTY)
210 () - (EMPTY)
211 () - (EMPTY)
212 () - (EMPTY)
213 () - (EMPTY)
214 () - (EMPTY)
215 () - (EMPTY)
216 () - (EMPTY)
217 () - (EMPTY)
218 () - (EMPTY)
219 () - (EMPTY)
220 () - (EMPTY)
221 () - (EMPTY)
222 () - (EMPTY)
223 () - (EMPTY)
224 () - (EMPTY)
225 () - (EMPTY)
226 () - (EMPTY)
227 () - (EMPTY)
228 () - (EMPTY)
229 () - (EMPTY)
230 () - (EMPTY)
231 () - (EMPTY)
232 () - (EMPTY)
233 () - (EMPTY)
234 () - (EMPTY)
235 () - (EMPTY)
236 () - (EMPTY)
237 () - (EMPTY)
238 () - (EMPTY)
239 () - (EMPTY)
240 () - (EMPTY)
241 () - (EMPTY)
242 () - (EMPTY)
243 () - (EMPTY)
244 () - (EMPTY)
245 () - (EMPTY)
246 () - (EMPTY)
247 () - (EMPTY)
248 () - (EMPTY)
249 () - (EMPTY)
250 () - (EMPTY)
251 () - (EMPTY)
252 () - (EMPTY)
253 () - (EMPTY)
254 () - (EMPTY)
255 () - (EMPTY)
256 () - (EMPTY)
257 () - (EMPTY)
258 () - (EMPTY)
259 () - (EMPTY)
260 () - (EMPTY)
261 () - (EMPTY)
262 () - (EMPTY)
263 () - (EMPTY)
264 () - (EMPTY)
265 () - (EMPTY)
266 () - (EMPTY)
267 () - (EMPTY)
268 () - (EMPTY)
269 () - (EMPTY)
270 () - (EMPTY)
271 () - (EMPTY)
272 () - (EMPTY)
273 () - (EMPTY)
274 () - (EMPTY)
275 () - (EMPTY)
276 () - (EMPTY)
277 () - (EMPTY)
278 () - (EMPTY)
279 () - (EMPTY)
280 () - (EMPTY)
281 () - (EMPTY)
282 () - (EMPTY)
283 () - (EMPTY)
284 () - (EMPTY)
285 () - (EMPTY)
286 () - (EMPTY)
287 () - (EMPTY)
288 () - (EMPTY)
289 () - (EMPTY)
290 () - (EMPTY)
291 () - (EMPTY)
292 () - (EMPTY)
293 () - (EMPTY)
294 () - (EMPTY)
295 () - (EMPTY)
296 () - (EMPTY)
297 () - (EMPTY)
298 () - (EMPTY)
299 () - (EMPTY)
300 () - (EMPTY)
301 () - (EMPTY)
302 () - (EMPTY)
303 () - (EMPTY)
304 () - (EMPTY)
305 () - (EMPTY)
306 () - (EMPTY)
307 () - (EMPTY)
308 () - (EMPTY)
309 () - (EMPTY)
310 () - (EMPTY)
311 () - (EMPTY)
312 () - (EMPTY)
313 () - (EMPTY)
314 () - (EMPTY)
315 () - (EMPTY)
316 () - (EMPTY)
317 () - (EMPTY)
318 () - (EMPTY)
319 () - (EMPTY)
320 () - (EMPTY)
321 () - (EMPTY)
322 () - (EMPTY)
323 () - (EMPTY)
324 () - (EMPTY)
325 () - (EMPTY)
326 () - (EMPTY)
327 () - (EMPTY)
328 () - (EMPTY)
329 () - (EMPTY)
330 () - (EMPTY)
331 () - (EMPTY)
332 () - (EMPTY)
333 () - (EMPTY)
334 () - (EMPTY)
335 () - (EMPTY)
336 () - (EMPTY)
337 () - (EMPTY)
338 () - (EMPTY)
339 () - (EMPTY)
340 () - (EMPTY)
341 () - (EMPTY)
342 () - (EMPTY)
343 () - (EMPTY)
344 () - (EMPTY)
345 () - (EMPTY)
346 () - (EMPTY)
347 () - (EMPTY)
348 () - (EMPTY)
349 () - (EMPTY)
350 () - (EMPTY)
351 () - (EMPTY)
352 () - (EMPTY)
353 () - (EMPTY)
354 () - (EMPTY)
355 () - (EMPTY)
356 () - (EMPTY)
357 () - (EMPTY)
358 () - (EMPTY)
359 () - (EMPTY)
360 () - (EMPTY)
361 () - (EMPTY)
362 () - (EMPTY)
363 () - (EMPTY)
364 () - (EMPTY)
365 () - (EMPTY)
366 () - (EMPTY)
367 () - (EMPTY)
368 () - (EMPTY)
369 () - (EMPTY)
370 () - (EMPTY)
371 () - (EMPTY)
372 () - (EMPTY)
373 () - (EMPTY)
374 () - (EMPTY)
375 () - (EMPTY)
376 () - (EMPTY)
377 () - (EMPTY)
378 () - (EMPTY)
379 () - (EMPTY)
380 () - (EMPTY)
381 () - (EMPTY)
382 () - (EMPTY)
383 () - (EMPTY)
384 () - (EMPTY)
385 () - (EMPTY)
386 () - (EMPTY)
387 () - (EMPTY)
388 () - (EMPTY)
389 () - (EMPTY)
390 () - (EMPTY)
391 () - (EMPTY)
392 () - (EMPTY)
393 () - (EMPTY)
394 () - (EMPTY)
395 () - (EMPTY)
396 () - (EMPTY)
397 (end) -  The end. 
398 (altend) -  Alternative ending. 
399 (random) -  [RANDOM]Pick a destination at random[/RANDOM] from 400 or 397 or 398. 
400 (next) -  This is the next section. Go on to the end at 397. 
END DEBUG OUTPUT
