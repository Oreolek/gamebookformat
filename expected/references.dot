digraph gamebook {
1->400
1->399
 399->400
399->397
399->398
400->397
}
