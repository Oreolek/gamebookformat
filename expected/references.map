{"Dummy": 396, "end": 397, "random": 399, "altend": 398, "next": 400, "start": 1}