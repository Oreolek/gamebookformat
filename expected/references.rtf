{\rtf1\ansi\ansicpg1252\cocoartf1038\cocoasubrtf360
{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
{\colortbl;\red255\green255\blue255;}
\paperw11900\paperh16840\margl1440\margr1440\vieww14140\viewh14860\viewkind0
\pard\tx566\tx1133\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\ql\qnatural\pardirnatural

\f0\b\fs24 \cf0

\b \qc Gamebook
\b0\
\
\b \qc Introduction
\b0\



\ql  This gamebook demonstrates simple references between sections. Also notice that an intro section like this one can reference sections, like the start at \b {\field{\*\fldinst  REF s1 \\h }{\fldrslt 1}}
\b0
 or the next section at \b {\field{\*\fldinst  REF s400 \\h }{\fldrslt 400}}
\b0
. References can include an optional name that is used in some formats, so the same links could also be rendered as \b {\field{\*\fldinst  REF s1 \\h }{\fldrslt start}}
\b0

 and \b {\field{\*\fldinst  REF s400 \\h }{\fldrslt next}}
\b0

.  \
\
\b Turn to 1 to begin.
\b0\
\
\b \qc {\*\bkmkstart s1}1{\*\bkmkend s1}
\b0\
\ql  This is where the adventure begins. You can go on to the next section, see \b {\field{\*\fldinst  REF s400 \\h }{\fldrslt 400}}
\b0
 or try the random selection feature at \b {\field{\*\fldinst  REF s399 \\h }{\fldrslt 399}}
\b0
.  \
\

\b \qc {\*\bkmkstart s397}397{\*\bkmkend s397}
\b0\
\ql  The end.  \
\

\b \qc {\*\bkmkstart s398}398{\*\bkmkend s398}
\b0\
\ql  Alternative ending.  \
\

\b \qc {\*\bkmkstart s399}399{\*\bkmkend s399}
\b0\
\ql  Pick a destination at random from \b {\field{\*\fldinst  REF s400 \\h }{\fldrslt 400}}
\b0
 or \b {\field{\*\fldinst  REF s397 \\h }{\fldrslt 397}}
\b0
 or \b {\field{\*\fldinst  REF s398 \\h }{\fldrslt 398}}
\b0
.  \
\

\b \qc {\*\bkmkstart s400}400{\*\bkmkend s400}
\b0\
\ql  This is the next section. Go on to the end at \b {\field{\*\fldinst  REF s397 \\h }{\fldrslt 397}}
\b0
.  \
\

}
