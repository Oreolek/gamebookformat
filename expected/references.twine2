<tw-storydata name="Gamebook" startnode="1" creator="Twine" creator-version="2.0" format="Harlowe" options="">
<style role="stylesheet" id="twine-user-stylesheet" type="text/twine-css"></style><script role="script" id="twine-user-script" type="text/twine-javascript"></script>
<tw-passagedata pid="1" name="1" tags="" position="10,10"> This is where the adventure begins. You can go on to the next section, see [[400]] or try the random selection feature at [[399]]. </tw-passagedata>
<tw-passagedata pid="397" name="397" tags="" position="3970,3970"> The end. </tw-passagedata>
<tw-passagedata pid="398" name="398" tags="" position="3980,3980"> Alternative ending. </tw-passagedata>
<tw-passagedata pid="399" name="399" tags="" position="3990,3990"> Pick a destination at random from [[400]] or [[397]] or [[398]]. </tw-passagedata>
<tw-passagedata pid="400" name="400" tags="" position="4000,4000"> This is the next section. Go on to the end at [[397]]. </tw-passagedata>
</tw-storydata>
