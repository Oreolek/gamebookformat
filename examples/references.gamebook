max = 400
* 1 start
This is where the adventure begins. You can go on to the
next section, see [[next]] or try the other instead, see [[other]].

* next
This is the next section. Go on to the end at [[end]].

* other
This is another section. You can try the next section now,
see [[next]], or go on to the end, see [[end]].

* end
The end.

