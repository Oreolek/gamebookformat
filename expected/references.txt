    Gamebook

Introduction
 This gamebook demonstrates simple references between sections. Also notice that an intro section like this one can reference sections, like the start at 1 or the next section at 400. References can include an optional name that is used in some formats, so the same links could also be rendered as start (1) and next (400). 


Turn to 1 to begin.
1
 This is where the adventure begins. You can go on to the next section, see 400 or try the random selection feature at 399. 

397
 The end. 

398
 Alternative ending. 

399
 Pick a destination at random from 400 or 397 or 398. 

400
 This is the next section. Go on to the end at 397. 

